packages/chart/src/Bubble.ts

```diff
@@ -65,13 +66,21 @@ export class Bubble extends SVGWidget {
         if (this.useClonedPalette()) {
             this._palette = this._palette.cloneNotExists(this.paletteID() + "_" + this.id());
         }
-
+        console.log('this.cloneData()',this.cloneData());
         const root = d3Hierarchy({ children: this.cloneData() })
             .sum(function (d) { return d[1]; })
             .sort(function (b, a) {
                 return a[1] < b[1] ? -1 : a[1] > b[1] ? 1 : 0;
             })
             ;
+        if(this.minRadius() !== null && this.maxRadius() !== null){
+            const interp = d3Interpolate(this.minRadius(),this.maxRadius());
+            const minVal = Math.min.apply(undefined, root.children.map(n=>n.value));
+            const maxVal = Math.max.apply(undefined, root.children.map(n=>n.value));
+            this.d3Pack.radius(function(n){
+                return n.value === minVal ? context.minRadius() : interp((n.value - minVal)/(maxVal - minVal));
+            });
+        }
```

Comment:
Should be able to specify a min on its own (without a max) and similarly for max?  This would make it the same as yAxisDomainLow and yAxisDomainHigh in the XYAxis charts?

Reply:
If the minimum radius would have been 50 for example, but the user specifies the maxRadius as 40 (but doesn't specify minRadius), should all of the circles have a radius of 40 (essentially saying "Don't allow them to get any bigger than 40")? Or should it set the largest value's radius at 40 and relatively scale the rest of them (essentially saying "Use 40 for the largest and scale the rest")?

Reply:
...because if I'm remembering correctly the yAxisDomainLow/yAxisDomainHigh simply cap the upper and lower ticks, but the data still displays beyond the Low and/or High (they just get clipped off at the edge of the widget).

Reply:
Ahh ok - Its the difference between domain low/high (I know all exam results must be between 0 and 100) and physical pixel min and max...
But when you set the constrained min/max doesn't that prevent the bubble chart from using up all the available space?